.travis/config.sh

```diff
@@ -0,0 +1,13 @@
+export LSR_MOLECULE_DEPS='-rmolecule_requirements.txt'
```

Comment:
It would be nice to add some copyright information to all code files to simplify license audits, for example a `# SPDX-License-Identifier: ...` header with the chosen license.